public/style.css

```diff
@@ -677,55 +698,95 @@ form button {
     flex: 1 100%;
   }
 
-  .closeBtn {
-    display: none;
-    padding: 10px;
-    margin: 0;
+  .item:nth-of-type(1) {
+    background-color: var(--white);
+    background-image: url(./images/imgbg.jpg);
+    background-repeat: no-repeat;
+    padding: 260px 15% 0 15%;
+    border-bottom-left-radius: 100px;
   }
 
-  .closeBtn img {
-    width: 15px;
-    float: right;
+  .achievement {
+    border: 1px solid var(--collapseborder);
+    width: 80.28%;
+    height: 500px;
+    margin: 100px auto;
+    border-radius: 25px;
+    padding: 24px;
+    background-color: var(--white);
   }
 
-  .popup {
-    width: 96%;
-    height: 96vh;
+  .achievement .fullPic {
+    width: 50%;
+    float: left;
+    background: green;
+    padding: 10px;
+    margin: 24px;
+  }
+
+  /* popup css */
+
+  /* debut section 2 */
+  .bg-popup {
+    width: 100%;
+    height: 100vh;
+    background-color: rgba(0, 0, 0, 0.6);
     position: fixed;
-    background-color: var(--white);
     overflow: hidden;
-    top: 2vh;
-    left: 2%;
-    z-index: 2;
-    border: 2px solid var(--collapseborder);
-    box-shadow: 0 0 10px 5px silver;
+    z-index: 1;
+    top: 0;
+  }
+
+  .popup {
+    width: 45%;
+    height: 95%;
+    background-color: white;
+    margin: 1.5% auto;
     border-radius: 8px;
-    padding: 24px;
-    margin: auto;
+    padding: 25px;
     display: flex;
+    flex-wrap: wrap;
   }
 
-  .achievement {
-    border: 1px solid var(--collapseborder);
-    width: 80.28%;
-    margin: 100px auto;
-    border-radius: 25px;
-    background-color: var(--white);
+  .popup .fullPic {
+    border-radius: 8px;
+    width: 100%;
+  }
+
+  .popup .step:nth-of-type(3) {
+    flex: 9;
+    order: 1;
   }
 
-  .aside {
-    flex: 1 0 0;
+  .popup .step:nth-of-type(2) {
+    flex: 1;
+    order: 2;
   }
 
-  .achievement .aside-1 {
+  .popup p,
+  .popup .step:nth-of-type(4) {
+    flex: 1;
+  }
+
+  .popup .step:nth-of-type(1) {
+    flex: 1 100%;
+    padding: 10px;
     border-radius: 8px;
-    margin: 24px;
+    background-color: green;
+    order: 3;
   }
 
-  .achievement .aside-2 {
-    margin: 24px;
+  .popup .paragraph {
+    order: 3;
   }
 
+  .popup .step:nth-of-type(4) {
+    order: 5;
+    padding-top: 5px;
+  }
+
+  /* fin popup */
+
```

Comment:
Please make sure to stick to the design template as much as possible.  Your design isn't matching that of the Figma template and below are the changes you can make to stick to the design:
- [ ] Your model is missing the `See live` and `See source` buttons.  Kindly consider adding the buttons as it will be a good user experience for your users to have a button to see a live version of the project using showcasing on your portfolio and also be able to go to the project source to read more able how you built the project.

- [ ] The layout of the description, stacks, and buttons below the model image should be changed to align with that of the Figma design. This issue can be fixed by using CSS `grid` to layout properly and responsively to create a good user experience. See below for more details:
Figma:
<img width="626" alt="Screenshot 2023-04-20 at 21 27 42" src="https://user-images.githubusercontent.com/107192076/233631642-fd183bda-4f6c-498f-a5bc-7b87efeac40c.png">

Your design:
<img width="695" alt="Screenshot 2023-04-21 at 11 36 02" src="https://user-images.githubusercontent.com/107192076/233631666-4680893b-0cc6-4b2b-b563-f4d511a7d8d7.png">